content/en/docs/developer_guide/flows/flow_triggers/oas_flow_trigger.md

```diff
@@ -81,15 +84,29 @@ If the **Body** is a "raw" `string` or `Buffer`, then no additional processing w
 
 If the **Body** is a type _other_ than `string` or `Buffer` (e.g. an `Object` or `Array`), then **OpenAPI** flow-trigger may automatically JSON encode the response if there is exactly one response media type that is JSON (e.g. "application/json"). Otherwise, the response content encoding is ambiguous, and results in a `500 Internal Server Error`. In this case, you should ensure you encode the **Body** within the flow and set an appropriate `content-type` header in **Headers** that matches the encoding for the **Body**.
 
-{{% alert title="Note" color="primary" %}}
-Currently, the response body is not validated against the JSON schema, so it is possible to send _invalid_ responses. This will be improved in a future release.
-{{% /alert %}}
+Additional validation is also performed to ensure the response body matches what's defined in your specification. Any mismatches will result in a `500 Internal Server Error`. 
+
+#### Required body
+If the specification documents content for the response, then this indicates a body should be returned. Similarly, if no content is documented, a body should not be returned.
+
+Note: in OpenAPI 2, responses have a `schema` property which is used to document that a response has content.
```

Comment:
This reads a bit funny... "specification documents content"...  "if no content is documented".  I'm not sure if users understand that `content` is a [Response Object content](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#responseObject).  Maybe add links to OAS.  Maybe YAML examples of required vs. no-content?

Also, the "Note: in OpenAPI 2", should be "2.0", but do we need the "Note: "?

Reply:
I guess that's another element of confusion. `content` isn't actually meaning `Response object content` here. It's content as in "204 No Content". I was wondering if there's some cases where "content" means something a bit more than "Response body" (since they say content a lot in the OpenAPI spec) but it seems to be equivalent